editor-integration/emacs/dune.el

```diff
@@ -0,0 +1,54 @@
+;;; dune.el --- Align words in an intelligent way
+
+;; Copyright 2018 Jane Street Group, LLC <opensource@janestreet.com>
+;; URL: https://github.com/ocaml/dune
+;; Version: 1.0
+
+;;; Commentary:
+
+;; This package provides helper functions for interacting with the
+;; dune build system from emacs.
+
+;; Installation:
+;; You need to install the OCaml program ``dune''.  The
+;; easiest way to do so is to install the opam package manager:
+;;
+;;   https://opam.ocaml.org/doc/Install.html
+;;
+;; and then run "opam install dune".
+
+;;; Code:
+
+(defgroup dune nil
+  "Integration with the dune build system."
+  :tag "Dune build system."
+  :version "1.0"
```

Comment:
Should we `subst` that if this is tracked within the dune repository?

Reply:
This version is for melpa, the emacs package manager. It needs to be in plain text in the git repository (melpa will fetch it directly from github). It also needs to be independent of the main dune version, otherwise this will cause a lot of spurious upgrades for emacs users.